dist/images/ovnkube.sh

```diff
@@ -282,10 +303,10 @@ check_health () {
     "ovnkube"|"ovnkube-master")
     ;;
     "ovnnb_db"|"ovnsb_db")
-    ctl_file=/var/run/openvswitch/${1}.ctl
+    ctl_file=${OVN_RUNDIR}/${1}.ctl
     ;;
     "ovn-northd"|"ovn-controller"|"ovsdb-server"|"ovs-vswitchd"|"ovn-nbctl")
-    ctl_file=/var/run/openvswitch/${1}.${2}.ctl
+    ctl_file=${OVN_RUNDIR}/${1}.${2}.ctl
```

Comment:
@numansiddique you will need to create a new `case statement` for ovsdb-server and ovs-vswitchd and use the control path in ${OVS_RUNDIR}

Reply:
Done